ops/jenkins/bfd-jenkins-sonarqube/Jenkinsfile

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env groovy
+
+// These variables are accessible throughout this file (except inside methods and classes).
+def verboseMaven = params.verbose_mvn_logging
+def appBuildResults
+
+pipeline {
+    agent {
+        kubernetes {
+            defaultContainer 'bfd-cbc-build'
+            yaml """
+apiVersion: v1
+kind: Pod
+spec:
+  serviceAccount: bfd
+  containers:
+  - name: bfd-cbc-build
+    image: 'public.ecr.aws/c2o1d8s9/bfd-cbc-build:jdk17-mvn3-tfenv3-latest'
+    command:
+    - cat
+    tty: true
+    imagePullPolicy: Always
+    resources:
+      requests:
+        memory: '16384Mi'
+        cpu: '8000m'
+      limits:
+        memory: '16384Mi'
+        cpu: '8000m'
+"""
+        }
+    }
+
+    parameters {
+        booleanParam(name: 'verbose_mvn_logging',
+        description: 'When true, `mvn` will produce verbose logs.',
+        defaultValue: false)
+        }
+    environment {
+        SONAR_TOKEN = credentials('svc-sonar-bfd')
+        SONAR_PROJECT_KEY = 'bfd-parent'
+        SONAR_HOST_URL = 'https://sonarqube.cloud.cms.gov'
+    }
+    stages {
+
+            stage('Prepare') {
+                steps {
+                    script {
+                        // Load the child Jenkinsfiles.
+                        scriptForApps = load('apps/build.groovy')
```

Comment:
Using `apps/build.groovy` might be reasonable here, but I'm not sure if we get everything from the build process that we want without extending that script... We may need to consult others on the team to understand the maven invocation to ensure that we get everything we want delivered to sonarqube, including new, more complete code coverage reports.